packages/shared/src/components/DevCardPlaceholder.tsx

```diff
@@ -56,15 +56,15 @@ export default function DevCardPlaceholder({
           id="devcard_placeholder_svg__a"
           width={2300}
           height={2030}
-          xlinkHref={!isLocked && profileImage}
```

Comment:
@rebelchris  Why did you have to remove the condition? I think in the design it should not be displayed.

Reply:
![Uploading Screenshot 2022-02-05 at 12.22.33.png…]()

See here :)

Reply:
Awesome. Then that should cover it 🚢